meetings/2023-06-29.md

```diff
@@ -0,0 +1,130 @@
+# W3C Solid Community Group: Notifications Panel
+
+* Date: 2023-06-29T14:00:00Z
+* Call: https://meet.jit.si/solid-notifications
+* Chat: https://gitter.im/solid/notifications-panel
+* Repository: https://github.com/solid/notifications-panel
+* Status: Published
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* elf Pavlik
+* [TallTed](https://github.com/TallTed) // Ted Thibodeau (he/him) (OpenLinkSw.com)
+* Hadrian Z
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/solid/specification/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* SC, eP
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+
+
+### Milestones
+URL: https://github.com/solid/notifications/milestone/1
+
+
+### DiscoveryClient and SubscriptionClient in Solid Notification Client
+URL: https://github.com/o-development/solid-notification-client/pull/1/files
+
+
+* eP: It doesn't implement features yet since they are optional. 
+* SC: Someone also had a client implementation in the feedback issue with the same name: https://github.com/woutslabbinck/Solid-Notification
+* eP: I'll take another look at it and will see if we can collaborate
+* SC: It seems to also implement DiscoveryClient and SubcriptionClient.
+* eP: Implementation in current PR only uses `text/turtle` since we don't require specific framing it doesn't seem to be possibly to just treat JSON-LD as plain JSON. If RDF has to parsed Turtle seems just fine.
```

Comment:
```suggestion
* eP: Implementation in current PR only uses `text/turtle`. Since we don't require specific framing, it doesn't seem to be possible to just treat JSON-LD as plain JSON. If RDF has to be parsed Turtle, seems just fine.
```